thirdparty/vars-ubuntu22-x86_64.sh

```diff
@@ -22,10 +22,10 @@
 #####################################################
 
 # jindosdk for Aliyun OSS
-JINDOSDK_DOWNLOAD="https://cdn-thirdparty.starrocks.com/jindosdk-4.6.8-linux-ubuntu22-x86_64.tar.gz"
-JINDOSDK_NAME="jindosdk-4.6.8-linux-ubuntu22-x86_64.tar.gz"
-JINDOSDK_SOURCE="jindosdk-4.6.8-linux-ubuntu22-x86_64"
-JINDOSDK_MD5SUM="52236053391091591c2d09684791e810"
+JINDOSDK_DOWNLOAD="https://jindodata-binary.oss-cn-shanghai.aliyuncs.com/release/6.5.0/jindosdk-6.5.0-linux-ubuntu22-x86_64.tar.gz"
```

Comment:
Can this URL be always accessed?

Reply:
assume stable from source:
https://github.com/aliyun/alibabacloud-jindodata/blob/master/docs/user/6.x/6.5.0/jindodata_download.md
but for security consideration and stability, a self hosted one can be safer(like supply chain attack)

Reply:
using this: https://cdn-thirdparty.starrocks.com/jindo/jindosdk-6.5.0-linux-ubuntu22-x86_64.tar.gz